dll.py

```diff
@@ -101,16 +175,32 @@ def insert_before(self, node: "Node", val: T):
         self._size += 1
 
     def remove(self, value: T):
-        """Remove node with given value if existing"""
+        """
+        Removes a node with the given value from the list
```

Comment:
Ook wel handig om te melden dat ie alleen de eerste occurence weghaald. En niks doet als er niet zo'n value inzit.

Reply:
De eerste occurence staat er al in (rl 181), dat andere zal ik er nog bij zetten